app/assets/stylesheets/helpers/_header.scss

```diff
@@ -109,14 +109,15 @@
       &.focus,
       &:focus {
         background: #fff;
+        outline-offset: -3px;
       }
 
       @include ie-lte(7){
         padding-left: 0;
       }
       .js-enabled & {
         width: 86%;
-        @include calc(width, "100% - 37px");
+        @include calc(width, "100% - 35.1px"); /* 35.1 is for Safari, which renders this input too narrow */
```

Comment:
It is, yes. Every other browser seems to understand an integer width correctly, but Safari for some reason drops a pixel. Setting it to 35.1px makes Safari over-compensate while other browsers round it down. It's weird, I know.